src/test_l1_dsHdmiIn.c

```diff
@@ -633,10 +659,12 @@ void test_l1_dsHdmiIn_positive_dsHdmiInSelectZoomMode(void) {
     UT_ASSERT_EQUAL(dsHdmiInInit(), dsERR_NONE);
 
     // Step 2: Loop through all dsVideoZoom_t values and call dsHdmiInSelectZoomMode()
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
Why is dsERR_NONE commented out here with the Enhanced error code?

Reply:
Removed the flag, it was added since the function implementation was not complete and  dsERR_NONE was missing in the HAL code. It is classified under other errors, so removed the flag.